### PR TITLE
 docs: added coming soon clarification to the landing pages for incomplete classifiers

### PR DESCRIPTION
**Pull request by Jazmin915**
I added a "coming soon" message to each landing page of a classifier we have not worked on yet. I also linked our discord in the message.

### CODE REVIEW

**Review by rorleth — CHANGES_REQUESTED** [review]
As opposed to adding the very same content in 8 files, I'd suggest you put the "coming soon" text in a single file and include that. See https://docusaurus.io/docs/markdown-features/react#importing-markdown for how to do that. 

Bonus points if you put the discord link into a component as well and then use that in all the places where we point to discord like in the header, so if we ever change the invite, we have a single place to change it in the source (DRY principle - do not repeat yourself) and not risk forgetting one or two.

**Comment by Jazmin915** [comment]
Im still going to try and get the discord link to be a component but so far I have the "coming soon message" imported from its own file

File: docsite/sidebars.ts [hunk 1]
```diff
@@ -12,7 +12,7 @@ import type {SidebarsConfig} from '@docusaurus/plugin-content-docs';
  */
 const sidebars: SidebarsConfig = {
   // By default, Docusaurus generates a sidebar from the docs folder structure
-  tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
+  tutorialSidebar: [{type: 'autogenerated', dirName: '.', }],
```
Comment (rorleth) [inline docsite/sidebars.ts:15]: Superfluous comma - I am kinda surprised this didn't barf when building.

**Comment by Jazmin915** [comment]
Removed the Superfluous comma

**Review by rorleth — APPROVED** [review]

**Pull request merged**